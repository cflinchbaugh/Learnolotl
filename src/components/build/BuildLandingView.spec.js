import React from 'react';
import BuildLandingView from './BuildLandingView';
import renderer from 'react-test-renderer';

it('renders and matches snapshot', () => {
    const tree = renderer
        .create(<BuildLandingView />)
        .toJSON();
        
    expect(tree).toMatchSnapshot();
});