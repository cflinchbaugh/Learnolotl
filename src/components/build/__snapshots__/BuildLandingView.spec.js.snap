// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and matches snapshot 1`] = `
<div
  className="build-landing-view-wrapper clearfix sc-bwzfXH ivtJYY"
>
  <p>
    Create new Learnolotl cards here!
  </p>
  <p>
    You want to learn more than just the few sample cards that came with the app?  No problem! It's easy to make your own Learnolotl cards and share them with others.
  </p>
  <div
    className="navigation-button-wrapper sc-bdVaJa bSfISc"
  >
    <button
      className="navigation-button"
      onClick={undefined}
    >
      =&gt;
    </button>
  </div>
</div>
`;
